.github/workflows/release.yml

```diff
@@ -46,3 +46,18 @@ jobs:
         run: |
           make buildx.image
           make buildx.push
+
+  trivy:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: ${{ env.RELEASE_TAG }}
```

Comment:
```suggestion
          image-ref: openebs/linux-utils:${{ env.RELEASE_TAG }}
```

may need to check  if you can pass `image-ref` like this. But I think this should work.

Reply:
@akhilerm It worked.